### ISSUE TITLE
WLAN USB Adapter

### ISSUE THREAD

**Issue by JSAnyone** [avanc/photOS#68]
Hi,
Are WLAN USB Adapters like the famous [Edimax](https://www.reichelt.de/wlan-adapter-usb-150-mbit-s-edi-ew-7811un-v2-p293505.html?PROVID=2788&gclid=CjwKCAiAh_GNBhAHEiwAjOh3ZNRfchBbr-xRQyeZme72hhR_becVgLjn6Tv2rfI-r5MGT0ZNK3iP9RoCIrAQAvD_BwE&&r=1) supported?
They are easy to use on Debian, but it looks like it isn't that easy on buildroot:
- https://www.grappendorf.net/tutorials/nerves-pizero-edimax.html
- https://otm.my.id/2019/05/29/enabling-wi-fi-usb-dongle-in-buildroot-buildroot-part-3/

I want to use it on Raspberry Pi directly via USB, so the question isn't about the soldering but about the software part.

**Comment by JSAnyone** [comment]
I tried some things out and realized that it is already possible without compiling a new image:

1. Flash the image and add wifi credentials like usually
2. Boot your Raspberry Pi with a ethernet connection
2. Start a SSH connection to the Raspberry Pi
3. Log in with user `root` and empty password
4. Remount filesystem: Type `mount -o rw,remount /`
5. Open modules file: Type `vi /etc/modules`
6. Edit modules file: Type `i` to start insert mode, type `R8188eu`, press `esc` to leave insert mode, type `:wq` to save and close the file
7. Type `reboot` to restart photOS

That's all it took to get my Edimax WLAN Adapter working on a Raspberry Pi. 
If you think it might be interesting for others I can add it to the Wiki of this repository.

**Comment by avanc** [comment]
Reminder to myself to add this to the wiki. I didn't knew it as well :-)

**Comment by ueen** [comment]
> 7\. type `R8188eu`, press `esc` to leave insert mode,

@jscmidt please clarify, did you override something with this or should one add it in a new line? 
Merry Christmas 🎄

**Comment by jscmidt** [comment]
If I remember it correctly the file was empty, but you shouldn’t do anything wrong with just appending.

**Comment by ueen** [comment]
Awesome thank you for the quick reply, I appended but it didn't work unfortunately, I have the TP-Link TL-WN725N and raspberry pi 1, it usually should be plug n play...

**Comment by jscmidt** [comment]
Unfortunately it isn’t plug and play, since as far as I know photOS is build on buildroot and not raspberry Pi OS, so drivers might be different…

**Comment by ueen** [comment]
True, we figured it out so far that we can set iwconfig via ssh manually and get a connection. 
But we can't persist it after reboot, the config file is not working or documented is to little for us
